### PR TITLE
Documentation: add example python api call for ExplainPlan.explainPotentialGpuPlan [skip ci]

### PR DESCRIPTION
**Pull request by tgravescs**
fixes https://github.com/NVIDIA/spark-rapids/issues/4745

### CODE REVIEW

**Comment by tgravescs** [comment]
build

**Comment by tgravescs** [comment]
build

File: docs/get-started/getting-started-workload-qualification.md [hunk 1]
```diff
@@ -188,7 +188,20 @@ which is the same as the driver logs with `spark.rapids.sql.explain=all`.
...
+   val output=com.nvidia.spark.rapids.ExplainPlan.explainPotentialGpuPlan(df)
+   ```
+
+   Python API:
+   ```python
+   output = sparkSession._jvm.com.nvidia.spark.rapids.ExplainPlan.explainPotentialGpuPlan(df._jdf, "ALL")
+   ```
+     - First parameter, `df` is the Spark DataFrame to get the query plan from.
+     - Second parameter, `ALL` returns all the explain data, you can change to `NOT_ON_GPU` to return just what
+       does not work on the GPU. The Scala API, ALL is the default when second parameter is not specified.
```
Comment (jlowe) [inline docs/get-started/getting-started-workload-qualification.md:202]: "The Scala API," is a bit confusing here, was it intended to say "For the Scala API," or was this supposed to be in a separate section like "Python API:" above?

Comment (tgravescs) [reply docs/get-started/getting-started-workload-qualification.md]: really I just want the java doc of this function which we don't publish so I changed this to just have a function documentation section

**Comment by tgravescs** [comment]
IO need to fix a few formatting things

**Comment by tgravescs** [comment]
ok formatting is fixed

**Comment by tgravescs** [comment]
build

File: docs/get-started/getting-started-workload-qualification.md [hunk 2]
```diff
@@ -145,6 +145,47 @@ which is the same as the driver logs with `spark.rapids.sql.explain=all`.
...
+```
+
+Looks at the CPU plan associated with the dataframe and outputs information
+about which parts of the query the RAPIDS Accelerator for Apache Spark
+could place on the GPU. This only applies to the initial plan, so if running
+with adaptive query execution enable, it will not be able to show any changes
+in the plan due to that.
+
+This is very similar output you would get by running the query with the
+Rapids Accelerator enabled and with the config `spark.rapids.sql.enabled` enabled.
```
Comment (jlowe) [inline docs/get-started/getting-started-workload-qualification.md:161]:
```suggestion
RAPIDS Accelerator enabled and with the config `spark.rapids.sql.enabled` enabled.
```

**Comment by tgravescs** [comment]
build

**Review by jlowe — APPROVED** [review]

**Pull request merged**